packages/vite-plugin-cloudflare/e2e/helpers.ts

```diff
@@ -46,7 +46,9 @@ export const test = baseTest.extend<{
 			debuglog("Fixture copied to " + projectPath);
 			await updateVitePluginVersion(projectPath);
 			debuglog("Updated vite-plugin version in package.json");
-			runCommand(`${pm} install`, projectPath);
+			// Note: we use --force because otherwise yarn
+			//       gets stuck during installation in CI
+			runCommand(`${pm} install --force`, projectPath);
```

Comment:
Note: alternatively I could set the flag conditionally based on `process.env.CI`, but I figured we might as well keep the same behavior locally to avoid potential confusion later on (e.g. if for example something breaks when `--force` is used)